AppCenterCrashes/AppCenterCrashes/MSCrashes.mm

```diff
@@ -1096,7 +1096,13 @@ + (void)sendErrorAttachments:(NSArray<MSErrorAttachmentLog *> *)errorAttachments
 
 - (void)notifyWithUserConfirmation:(MSUserConfirmation)userConfirmation {
   NSArray<MSErrorAttachmentLog *> *attachments;
-
+  
+  // Check if there is no handler set and unprocessedReports are not initialized as NSMutableArray (Init occurs in correct call sequence).
+  if ([MSCrashes sharedInstance].userConfirmationHandler==nil && !self.unprocessedReports) {
+    MSLogError(MSCrashes.logTag, @"Incorrect implementation of notifyWithUserConfirmation: should only be called from userConfirmationHandler. For more information refer to online documentation.");
```

Comment:
implementation -> usage
should -> it should
online documentation -> the documentation